test/core/Application.js

```diff
@@ -48,4 +48,44 @@ describe('PIXI.Application', function ()
             done();
         });
     });
+
+    it('should be able to destroy filter manager gracefully', function (done)
+    {
+        const app = new PIXI.Application();
+
+        document.body.appendChild(app.view);
+
+        const sprite = new PIXI.Sprite(PIXI.Texture.WHITE);
+
+        sprite.position.x = 60;
+        sprite.position.y = 60;
+
+        app.stage.addChild(sprite);
+
+        let iterations = 3;
```

Comment:
Does this error reproduce with less than 3 iterations of applying a filter? Do they need to be applied asynchronously? What about?
```js
sprite.filters = [filter];
app.render();
sprite.filters = [filter];
app.render();
sprite.filters = [filter];
app.render();
```

Reply:
I've tried to use `ticker` – but no errors.